### DISCUSSION TITLE
Doesn't work when converted to monorepo

### DISCUSSION THREAD

**Discussion by mohsinamjad** [NarHakobyan/awesome-nest-boilerplate#71] [Q&A] [answered]
when the boilerplate is converted to monorepo, 
it crashes on everything most probably due to overridden commands in package.json as nest cli is not being used for starting or building an application,
it'd be great if there is a separate branch for monorepo.

**Comment by NarHakobyan** [comment]
Hi @mohsinamjad, to be honest, we never used nest-cli to run our projects 😄, but anyway I agree with you that we need to support cli too.

**Comment by NarHakobyan** [comment, marked as answer]
Nest-cli works now
